code/clue.js

```diff
@@ -41,19 +260,100 @@ const randomSelector = array => {
 // With a killer, a weapon and a room.
 // The values will be set later.
 
+const mystery = {
+  killer: null,
+  weapon: null,
+  room: null
```

Comment:
Very nice you initialised mystery object with null values! That's the cleanest approach :)